### PR TITLE
Updating Readme.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by aman7870**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+│   ├── wallet.css
+├── js                                 All the working logic goes here                            
+│   ├── wallet.js                      
+├── tasks                              For assigning tasks to the members HTML page
+│   ├── create.html
+├── contribution.md                    CONTRIBUTION file for the repository
+├── index.html                         For Homepage HTML file    
+├── main.css                           All the styling goes here except wallet page
+├── readme.md                          README file for the repository
+├── script.js                          All the working logic goes here except wallet page
```
Comment (swarajpure) [inline CONTRIBUTING.md:18]: CONTRIBUTING.md
README.md

Comment (swarajpure) [reply CONTRIBUTING.md:18]: Spelling and case I meant

Comment (aman7870) [reply CONTRIBUTING.md:18]: Opps! I'm correcting it

**Review by swarajpure — APPROVED** [review]
LGTM! 👌🏻

**Pull request merged**